corehq/apps/export/templates/export/partial/export_download_progress.html

```diff
@@ -52,8 +52,19 @@
                                 <span ng-show="!isMultimediaDownload">{% trans "Collecting Data" %}</span>
                                 <span ng-show="isMultimediaDownload">{% trans "Collecting Multimedia" %}</span>
                         </button>
+                        {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
+                        <button type="button"
+                                class="btn btn-primary btn-full-width"
```

Comment:
`btn-primary` is only for a single primary action on a page. I'd switch this to `btn-default`.